runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -176,10 +182,21 @@ public long append(@Nonnull Set<UUID> streamIDs, @Nonnull Object object,
                         conflictInfo,
                         tokenResponse.getConflictKey(),
                         AbortCause.NEW_SEQUENCER, // in the future, perhaps define a new AbortCause?
-                        TransactionalContext.getCurrentContext()
-                );
+                        TransactionalContext.getCurrentContext());
             }
         }
 
+        ILogData ld = new LogData(DataType.DATA, object);
```

Comment:
Describe what is happening here. Write failed, so we calculate the size of the write to return to the caller.